rll/enable-managed-login.sh

```diff
@@ -82,7 +82,7 @@ else
   bin_dir="/usr/local/bin"
 fi
 
-if ! $rsc rl10 actions | grep --ignore-case --quiet /rll/login/control >/dev/null 2>&1; then
+if ! $rsc rl10 actions | grep --ignore-case /rll/login/control >/dev/null 2>&1; then
```

Comment:
I think it would be better to just get rid of the error output from `rsc`:

```bash
if ! $rsc rl10 actions 2>/dev/null | grep --ignore-case --quiet /rll/login/control; then
```

`grep` shouldn't be outputting anything else anyway with `--quiet` so there isn't a need for the other redirections that were thre.